department-of-veterans-affairs/caseflow
# Test cases memorializing Judge algorithm testing for prod rollout

Connects #6447 

### Description
Added a set of test cases to memorialize test scenarios described in ticket.

### Acceptance Criteria 
- [ ] Test cases pass

### Testing Plan
1. Execute ../spec/models/judge_schedule_period_spec.rb
2. Verify all 10 tests go green
